js/signup.js

```diff
@@ -1,27 +1,75 @@
 const form = document.querySelector('.frame-input');
 const inputEmail = document.querySelector('.input-email input');
-const inputPassword = document.querySelector('.input-pw input');
-const inputPasswordCheck = document.querySelector('#check-pw input');
+const inputPassword = document.querySelector('#input-pw input');
+const inputPasswordCheck = document.querySelector('#input-pw-check input');
 const errorMsgEmail = document.querySelector('.error-email');
 const errorMsgPassword = document.querySelector('.error-pw');
 const errorMsgPasswordCheck = document.querySelector('.error-check-pw');
 const showPasswordBtn = document.querySelector('.show-pw-btn');
 const showPasswordCheckBtn = document.querySelector('.show-pw-check-btn');
 
-// 이메일 : test@codeit.com, 비밀번호 : codeit101 으로 로그인 시, '/folder' 페이지로 이동
-function checkIsUser(e) {
-  e.preventDefault();
-
-  if (inputEmail.value !== 'test@codeit.com') {
-    alert(`이메일을 확인해 주세요.`);
-    return;
+async function checkEmailRequest() {
+  const url = 'https://bootcamp-api.codeit.kr/api/check-email';
+  const checkEmail = {
+    email: inputEmail.value,
+  };
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify(checkEmail),
+    });
+    const responseStatus = response.status;
+    if (responseStatus == 200) {
+      alert(`사용 가능한 이메일입니다`);
+    }
+  } catch (err) {
+    alert(`이미 사용 중인 이메일입니다`);
+    console.log(err.message);
   }
-  if (inputPassword.value !== 'codeit101') {
-    alert(`비밀번호를 확인해 주세요.`);
-    return;
+}
+
+async function checkIsValidSignupFormRequest() {
+  const url = 'https://bootcamp-api.codeit.kr/api/sign-up';
+  const signupForm = {
+    email: inputEmail.value,
+    password: inputPassword.value,
+  };
+  try {
+    console.log(`유효성 검사중`);
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify(signupForm),
+    });
+    const responseStatus = response.status;
+    if (responseStatus === 200) {
+      window.location.href = 'folder.html';
+    }
+  } catch (err) {
+    alert(`유효한 회원가입 형식이 아닙니다`);
+    console.log(err.message);
   }
+}
+
+function checkIsValidSignupForm() {
```

Comment:
```suggestion
function checkIsValidSignupForm(e) {
```
여기 파라미터로 e를 받아줘야되는데 안받아주고 있습니다.
때문에 e.preventDefault 메서드가 동작하지 않아서 예측하신대로 동작하지 않았던 것입니다.